packages/mcp-client/cli.ts

```diff
@@ -4,28 +4,48 @@ import { stdin, stdout } from "node:process";
 import { join } from "node:path";
 import { homedir } from "node:os";
 import type { StdioServerParameters } from "@modelcontextprotocol/sdk/client/stdio.js";
+import type { ServerConfig } from "./src/types";
 import type { InferenceProvider } from "@huggingface/inference";
-import { ANSI } from "./src/utils";
+import { ANSI, urlToServerConfig } from "./src/utils";
 import { Agent } from "./src";
 import { version as packageVersion } from "./package.json";
 
 const MODEL_ID = process.env.MODEL_ID ?? "Qwen/Qwen2.5-72B-Instruct";
 const PROVIDER = (process.env.PROVIDER as InferenceProvider) ?? "nebius";
 const ENDPOINT_URL = process.env.ENDPOINT_URL ?? process.env.BASE_URL;
 
-const SERVERS: StdioServerParameters[] = [
-	{
-		// Filesystem "official" mcp-server with access to your Desktop
-		command: "npx",
-		args: ["-y", "@modelcontextprotocol/server-filesystem", join(homedir(), "Desktop")],
-	},
-	{
-		// Playwright MCP
-		command: "npx",
-		args: ["@playwright/mcp@latest"],
-	},
+const SERVERS: (ServerConfig | StdioServerParameters)[] = [
+	// {
+	// 	// Filesystem "official" mcp-server with access to your Desktop
+	// 	command: "npx",
+	// 	args: ["-y", "@modelcontextprotocol/server-filesystem", join(homedir(), "Desktop")],
+	// },
+	// {
+	// 	// Playwright MCP
+	// 	command: "npx",
+	// 	args: ["@playwright/mcp@latest"],
+	// },
+	// now using "--url https://evalstate-flux1-schnell.hf.space/gradio_api/mcp/sse"
 ];
 
+// Handle --url parameter: parse comma-separated URLs into ServerConfig objects
+const urlIndex = process.argv.indexOf("--url");
+if (urlIndex !== -1 && urlIndex + 1 < process.argv.length) {
+	const urlsArg = process.argv[urlIndex + 1];
+	const urls = urlsArg
+		.split(",")
+		.map((url) => url.trim())
+		.filter((url) => url.length > 0);
+
+	for (const url of urls) {
+		try {
+			SERVERS.push(urlToServerConfig(url));
+		} catch (error) {
+			console.error(`Error adding server from URL "${url}": ${error.message}`);
+		}
+	}
+}
```

Comment:
i think you can just use this: 

https://nodejs.org/api/util.html#utilparseargsconfig

Reply:
and i would still keep the ones above as default (the stdio ones) if nothing is passed